packages/cli-e2e/__tests__/orgConfig.specs.ts

```diff
@@ -0,0 +1,157 @@
+import {join} from 'path';
+import {CLI_EXEC_PATH, getConfig, getPathToHomedirEnvFile} from '../utils/cli';
+import {ProcessManager} from '../utils/processManager';
+import {Terminal} from '../utils/terminal/terminal';
+import {config} from 'dotenv';
+import {ensureDirSync} from 'fs-extra';
+import {foldersContainSimilarFiles} from '../utils/file';
+import PlatformClient from '@coveord/platform-client';
+import {getPlatformClient} from '../utils/platform';
+config({path: getPathToHomedirEnvFile()});
+
+describe('org:config', () => {
+  const testOrgId = process.env.TEST_ORG_ID!;
+  const {accessToken} = getConfig();
+  const snapshotProjectPath = join('snapshot-project');
+  const defaultTimeout = 10 * 60e3;
+  let processManager: ProcessManager;
+  let platformClient: PlatformClient;
+
+  const createNewTerminal = (
+    args: string[],
+    procManager: ProcessManager,
+    cwd: string,
+    debugName: string
+  ) => {
+    if (process.platform === 'win32') {
+      args.unshift('node');
+    }
+    return new Terminal(args.shift()!, args, {cwd}, procManager, debugName);
+  };
+
+  const previewChange = (targetOrg: string, procManager: ProcessManager) => {
+    const args: string[] = [
+      CLI_EXEC_PATH,
+      'org:config:preview',
+      `-t=${targetOrg}`,
+    ];
+
+    return createNewTerminal(
+      args,
+      procManager,
+      snapshotProjectPath,
+      'org-config-preview'
+    );
+  };
+
+  const pushToOrg = async (targetOrg: string, procManager: ProcessManager) => {
+    const args: string[] = [
+      CLI_EXEC_PATH,
+      'org:config:push',
+      '--skipPreview',
+      `-t=${targetOrg}`,
+    ];
+    const pushTerminal = createNewTerminal(
+      args,
+      procManager,
+      snapshotProjectPath,
+      'org-config-push'
+    );
+
+    await pushTerminal.when('exit').on('process').do().once();
+  };
+
+  const pullFromOrg = async (
+    targetOrg: string,
+    procManager: ProcessManager,
+    destinationPath: string
+  ) => {
+    const args: string[] = [
+      CLI_EXEC_PATH,
+      'org:config:pull',
+      `-t=${targetOrg}`,
+    ];
+
+    const pullTerminal = createNewTerminal(
+      args,
+      procManager,
+      destinationPath,
+      'org-config-pull'
+    );
+
+    await pullTerminal.when('exit').on('process').do().once();
+  };
+
+  beforeAll(async () => {
+    platformClient = getPlatformClient(testOrgId, accessToken);
+    processManager = new ProcessManager();
+  });
+
+  afterAll(async () => {
+    await processManager.killAllProcesses();
+  });
+
+  describe('org:config:preview', () => {
+    it(
+      'should preview the snapshot',
+      async () => {
+        const previewTerminal = previewChange(testOrgId, processManager);
+
+        const expectedOutput = [
+          'Extensions',
+          '\\+   1 to create',
+          'Fields',
+          '\\+   2 to create',
+          'Filters',
+          '\\+   1 to create',
+        ].join('\\s*');
```

Comment:
What does the final regexp looks like?

Reply:
That's not the output from the E2E test, but it looks something like that
![image](https://user-images.githubusercontent.com/12199712/127026005-e7bacc29-8619-440a-97a6-e0eb457ffa27.png)